config/crd/bases/infrastructure.cluster.x-k8s.io_nutanixclusters.yaml

```diff
@@ -300,7 +303,7 @@ spec:
                 type: boolean
             type: object
         type: object
-    served: true
+    served: false
```

Comment:
can u pls point to the docs for the the meaning of served for my understanding

Reply:
https://book.kubebuilder.io/reference/markers/crd.html

The `+kubebuilder:unserved` annotation in Kubebuilder is used to indicate that a particular version of a Custom Resource Definition (CRD) should not be served by the Kubernetes API server.

The Kubernetes API server will not allow creation or modification of resources using the unserved version. However, it can still manage resources that were created with that version if they are stored in etcd. This is useful for ensuring that no new resources are created with the deprecated version while still maintaining the ability to work with existing resources during a migration period.